recipes/libx264/all/conanfile.py

```diff
@@ -25,11 +25,15 @@ class LibX264Conan(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "bit_depth": [8, 10, "all"],
+        "with_opencl": [True, False],
+        "with_asm": [True, False]
```

Comment:
I would use `enable_opencl` and `enable_asm`, personally, since `with_*` is typically used for options controlling external dependencies on CCI. Matches the project option names better as well.

Reply:
Option ``"with_opencl": [True, False],`` already exists in other 6 recipes in ConanCenter, and ``enable_opencl`` only in 1, so better use ``with_opencl``